examples/example/App.tsx

```diff
@@ -289,7 +295,7 @@ export default function App() {
 
           <ResponsiveSquare />
           <Gradient
-            sx={{ height: 50, width: 50, my: '$3' }}
+            sx={{ height: 50, w: 50, my: '$3' }}
```

Comment:
Does width still work?
if it does then I would keep it as "width" not "w" to be more descriptive since you have height

Reply:
Sure, the `width` prop still works. `w` is an alias, not replacement.

Reply:
Jesus FUCKING Christ @Frosty21